### PR TITLE
Add Helm Chart

### PR DESCRIPTION
**Pull request by AleDore**

### CODE REVIEW

File: helm/values-prod01.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
+microservice-chart:
+  namespace: "payments"
```
Comment (pasqualedevita) [inline helm/values-prod01.yaml:2]:
```suggestion
  namespace: "messages"
```

**Review by pasqualedevita — CHANGES_REQUESTED** [review]

**Review by pasqualedevita — APPROVED** [review]

File: helm/values-beta.yaml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+      port: 9090
+    initialDelaySeconds: 90
+    failureThreshold: 6
+    periodSeconds: 10
+  service:
+    create: true
+    port: 9090
+  ingress:
+    create: true
+    host: "weubeta.reminder.internal.io.pagopa.it"
```
Comment (pasqualedevita) [inline helm/values-beta.yaml:26]:
```suggestion
    host: "weubeta.messages.internal.io.pagopa.it"
```

File: helm/values-prod01.yaml [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+      port: 9090
+    initialDelaySeconds: 90
+    failureThreshold: 6
+    periodSeconds: 10
+  service:
+    create: true
+    port: 9090
+  ingress:
+    create: true
+    host: "weuprod01.reminder.internal.io.pagopa.it"
```
Comment (pasqualedevita) [inline helm/values-prod01.yaml:26]:
```suggestion
    host: "weuprod01.messages.internal.io.pagopa.it"
```

**Review by pasqualedevita — CHANGES_REQUESTED** [review]

**Comment by AleDore** [comment]
@giporro: Can we add specific application properties on helm's values?

**Review by giporro — DISMISSED** [review]
- add application properties
- added variables for kafka implementation. please add secrets for enviroments

**Pull request merged**